front/src/features/tables/views/LogDetailsTable.tsx

```diff
@@ -3,7 +3,7 @@ import { Button } from "@/components/atoms/Button";
 import { CardAtom } from "@/components/atoms/CardAtom";
 import { useNavigation } from "@/hooks/useNavigation";
 import { LogData } from "@/types/logDisplayOptions";
-import { ColumnContentTags } from "./components/ColumnContentTags";
+import { ColumnContentTags } from "../components/ColumnContentTags";
```

Comment:
Curious why some of your imports are relative and others are absolute?

Reply:
I took the habit of writing the imports that are linked to the same feature as relative, and the ones which are not as absolute. Is this bad practice ?